tests/unit/fixtures/datasets/open_images_v4/classification/test-annotations-human-imagelabels.csv

```diff
@@ -0,0 +1,20 @@
+ImageID,Source,LabelName,Confidence

```

Comment:
@lm-lily Which file read this file?

Reply:
@iizukak 
If I remove this file, I get the following error:

```
===================================================== FAILURES =====================================================
__________________________________________ test_build_tfds_classification __________________________________________
[gw6] linux -- Python 3.6.3 /usr/local/pyenv/versions/3.6.3/envs/python3.6/bin/python

    def test_build_tfds_classification():
        environment.setup_test_environment()
    
        # Build TFDS Dataset
        config_file = "unit/fixtures/configs/for_build_tfds_classification.py"
>       run(config_file, overwrite=True)

unit/executor_tests/test_build_tfds.py:44: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
../blueoil/cmd/build_tfds.py:96: in run
    builder.download_and_prepare()
/usr/local/pyenv/versions/3.6.3/envs/python3.6/lib/python3.6/site-packages/tensorflow_datasets/core/api_utils.py:52: in disallow_positional_args_dec
    return fn(*args, **kwargs)
/usr/local/pyenv/versions/3.6.3/envs/python3.6/lib/python3.6/site-packages/tensorflow_datasets/core/dataset_builder.py:287: in download_and_prepare
    download_config=download_config)
/usr/local/pyenv/versions/3.6.3/envs/python3.6/lib/python3.6/site-packages/tensorflow_datasets/core/dataset_builder.py:937: in _download_and_prepare
    max_examples_per_split=download_config.max_examples_per_split,
/usr/local/pyenv/versions/3.6.3/envs/python3.6/lib/python3.6/site-packages/tensorflow_datasets/core/dataset_builder.py:793: in _download_and_prepare
    for split_generator in self._split_generators(dl_manager):
../blueoil/utils/tfds_builders/classification.py:60: in _split_generators
    num_shards=self._num_shards(dataset),
../blueoil/utils/tfds_builders/classification.py:77: in _num_shards
    total_size = sum(image.nbytes for image, _ in dataset)
../blueoil/utils/tfds_builders/classification.py:77: in <genexpr>
    total_size = sum(image.nbytes for image, _ in dataset)
../blueoil/datasets/open_images_v4.py:261: in __getitem__
    files, labels = self.files_and_annotations
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = <blueoil.utils.config.ClassificationDataset object at 0x7f8438094cf8>

    @property
    @functools.lru_cache(maxsize=None)
    def files_and_annotations(self):
        files = []
        annotations = []
>       with open(self.annotations_csv, 'r') as f:
E       FileNotFoundError: [Errno 2] No such file or directory: 'unit/fixtures/datasets/open_images_v4/classification/test-annotations-human-imagelabels.csv'

../blueoil/datasets/open_images_v4.py:247: FileNotFoundError

```

`../blueoil/datasets/open_images_v4.py` calls to this file.